src/pages/Vaults/VaultsOverview/components/VaultsTable/VaultsTable.tsx

```diff
@@ -0,0 +1,72 @@
+import { CurrencyExt, CurrencyIdLiteral } from '@interlay/interbtc-api';
+import Big from 'big.js';
+
+import { CoinPair, CTA, Table, TableProps } from '@/component-library';
+
+import { CoinPairWrapper, NumericValue, Wrapper } from './VaultsTable.style';
+
+const columns = [
+  { name: 'Vault Pair', uid: 'pair' },
```

Comment:
We should have column names in the locale files - not critical, but we should tidy this up soon (vault dashboard text too).